pdr_backend/lake_info/overview.py

```diff
@@ -288,3 +290,52 @@ def validate_lake_tables_no_duplicates(self) -> List[str]:
         duplicate_rows.write_csv("validate_duplicate_rows.csv")
 
         return violations
+
+    @enforce_types
+    def validate_no_unmatched_payouts(self) -> List[str]:
+        """
+        @description
+            validates that all payouts have been able to match with a prediction
+        """
+        violations: list[str] = []
+
+        # get all payouts
+        # we want to select all payouts
+        # we want to then join with predictions, and then filter for unmatched payouts
+        # as long as their timestamp is within first/last prediction['timestamp] rows
+        query_unmatched_payouts = f"""
+            WITH payouts AS (
+                SELECT
+                    ID,
+                    timestamp
+                FROM {Table.from_dataclass(Payout).table_name}
+            ), bronze_predictions AS (
+                SELECT
+                    ID,
+                    timestamp,
+                    max(timestamp) as max_timestamp,
+                    min(timestamp) as min_timestamp
+                FROM {Table.from_dataclass(BronzePrediction).table_name}
+                GROUP BY ID, timestamp
+            ),
+            unmatched_payouts AS (
+                SELECT
+                    payouts.ID,
+                    payouts.timestamp
+                FROM payouts
+                LEFT JOIN bronze_predictions
+                ON payouts.ID = bronze_predictions.ID
+                WHERE bronze_predictions.ID IS NULL
+                AND payouts.timestamp BETWEEN bronze_predictions.min_timestamp AND bronze_predictions.max_timestamp
+            )
+            select * from unmatched_payouts;
+        """
+
+        rows_df: pl.DataFrame = self.db.query_data(query_unmatched_payouts)
+
+        if rows_df is None or rows_df.shape[0] == 0:
+            logger.info("No unmatched payouts found in the lake.")
```

Comment:
I think you also need to add the violation into the list

Reply:
I'm also a bit confused since the lake validate command was removed from the help string, so do we only do validations in describe now?

Reply:
"I think you also need to add the violation into the list"
Nice catch!

Reply:
"I'm also a bit confused since the lake validate command was removed from the help string, so do we only do validations in describe now?"

This was removed with your DOT removal PR... I'll re-introduce it.

![image](https://github.com/user-attachments/assets/1c8411c4-08bf-4c8b-b5d8-ab8e427a984b)

Reply:
"I'm also a bit confused since the lake validate command was removed from the help string..."

Part was removed with your DOT removal PR... I don't know when the rest was removed... I'll reintroduce them.

![image](https://github.com/user-attachments/assets/1c8411c4-08bf-4c8b-b5d8-ab8e427a984b)

Reply:
reintroduced helper text and reorganized helper functions so everything is in logical/onboarding order